requirements.txt

```diff
@@ -2,5 +2,6 @@ boto3==1.9.99
 Click==7.0
 cligj==0.5.0
 requests==2.21.0
+requests-toolbelt==0.9.1
```

Comment:
We'll also need this in `setup.py` right?

Reply:
https://github.com/mapbox/tilesets-cli/blob/ee30e216a73501d13c48bf9c902ca38baa90b7e3/setup.py#L28